src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -39,11 +42,23 @@ object Boot extends App {
         swaggerConfig.getString("licenseUrl"))
       ))
 
-    val service = system.actorOf(RawlsApiServiceActor.props(swaggerService), "rawls-service")
+    val storageDir = new File(conf.getString("workspace.storageDir"))
+    storageDir.mkdirs()
+    val workspaceDAO = new FileSystemWorkspaceDAO(storageDir)
+    val service = system.actorOf(RawlsApiServiceActor.props(swaggerService, WorkspaceService.constructor(workspaceDAO)), "rawls-service")
 
     implicit val timeout = Timeout(5.seconds)
     // start a new HTTP server on port 8080 with our service actor as the handler
-    IO(Http) ? Http.Bind(service, interface = "0.0.0.0", port = 8080)
+    import scala.concurrent.ExecutionContext.Implicits.global
+    (IO(Http) ? Http.Bind(service, interface = "0.0.0.0", port = 8080)).onComplete {
+      case Success(Http.CommandFailed(failure)) =>
```

Comment:
Right, I forget the specifics but since zamboni was never intended to be receiving messages from akka actors it expected that call to block and respond immediately